content/articles/export-controls-us-china-chips.md

```diff
@@ -0,0 +1,47 @@
+---
+title: New US export regulations will deprive China of equipment for manufacturing high-end chips 
+description: 
+img: ![technology-3092486](https://user-images.githubusercontent.com/59807305/197054435-474c0746-924c-4dfc-b4fb-a5f514b12a72.jpg)
```

Comment:
This is wrong
```
img: ![technology-3092486](https://user-images.githubusercontent.com/59807305/197054435-474c0746-924c-4dfc-b4fb-a5f514b12a72.jpg)
```

This is correct. Only image URL is needed here.
```
img: https://user-images.githubusercontent.com/59807305/197054435-474c0746-924c-4dfc-b4fb-a5f514b12a72.jpg
alt: some image caption
```

Inside the article body we need the markdown to actually show the image
```
![image caption here](https://some-image-url.jpg)
```